src/lumigo_opentelemetry/dependencies/__init__.py

```diff
@@ -51,28 +50,23 @@ def _prepare_resource_attributes_for_marshalling(
     }
 
 
-def _report_to_saas(url: str, lumigo_token: str, data: str):
+def _report_to_saas(url: str, lumigo_token: str, data: str) -> None:
     parsed_url = urlparse(url)
     timeout = environ.get("LUMIGO_CONNECTION_TIMEOUT", DEFAULT_CONNECTION_TIMEOUT)
 
-    try:
-        connection = HTTPSConnection(parsed_url.hostname, timeout=timeout)
-        connection.request(
-            "POST",
-            parsed_url.path,
-            data,
-            {
-                "Authorization": f"LumigoToken {lumigo_token}",
-                "Content-type": "application/json",
-            },
-        )
+    connection = HTTPSConnection(parsed_url.hostname or "", timeout=float(timeout))
+    connection.request(
+        "POST",
+        parsed_url.path,
+        data,
+        {
+            "Authorization": f"LumigoToken {lumigo_token}",
+            "Content-type": "application/json",
+        },
+    )
 
-        response = connection.getresponse()
-        response_status = response.status
+    response = connection.getresponse()
+    response_status = response.status
 
-        if response_status != 200:
-            logger.debug(
-                "Dependency report failed with status code %s", response_status
-            )
-    except Exception as e:
-        logger.debug("Dependency report failed", e)
+    if response_status != 200:
+        raise Exception(f"Dependency report failed with status code {response_status}")
```

Comment:
Can you add here more data? Maybe something like `response.text`.
In case of errors, it will make debugging easier.

Reply:
I think it is because of https://github.com/lumigo-io/opentelemetry-python-distro/blob/6f310a6e1357896c86c2ba253bcdb4659ca705d1/src/lumigo_opentelemetry/__init__.py#L169, which we need for `autowrapt` to work

Reply:
Done in b6b3e2d